api/internal/localizer/util.go

```diff
@@ -14,6 +14,9 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/filesys"
 )
 
+const DstPrefix = "localized"
+const LocalizeDir = "localized-files"
```

Comment:
can you add docstring for these two public variables?

Reply:
done